config/samples/backends/ceph/backend.yaml

```diff
@@ -0,0 +1,60 @@
+# Sample using Ceph for glance, cinder volume, and cinder backup.
+# Requires a running Ceph cluster and its `/etc/ceph` files in secret `ceph-conf-files`
+# This can be achieved with the `ceph` target of `install_yamls`
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
+spec:
+  extraMounts:
+    - name: v1
+      region: r1
+      extraVol:
+        - propagation:
+          - CinderVolume
+          - CinderBackup
+          - GlanceAPI
+          extraVolType: Ceph
+          volumes:
+          - name: ceph
+            projected:
+              sources:
+              - secret:
+                  name: ceph-conf-files
+          mounts:
+          - name: ceph
+            mountPath: "/etc/ceph"
+            readOnly: true
+  cinder:
+    template:
+      cinderVolumes:
+        ceph:
+          replicas: 1
+          customServiceConfig: |
+            [ceph]
```

Comment:
Unless/until [1] merges, with zed container images it will be necessary to specify the rbd_secret_uuid (the cluster's FSID) for nova. As this is just a sample file, should it include a comment that it's the user's responsibility to add that?

[1] https://review.opendev.org/c/openstack/cinder/+/878168

Reply:
I was planning to wait until we have nova working with cinder and then update the example to use Antelope containers.